app/assets/javascripts/home-ux.js

```diff
@@ -73,8 +74,10 @@ $(document).ready(function () {
         },
       });
     },
-    onChange: function () {
+    onChange: function (text) {
       if (isNaN(parseInt($("#select-beast-landsearch").val()))) {
+        $("#new-firstname-input").val(text.split(" ")[0]);
```

Comment:
This resets the new client form firstname and lastname inputs. Could have made a different PR for this. Leaving since I'm the only Author.